packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -1,115 +1,135 @@
+              @mouseleave="$emit('mouseleave:button', '')"
+              @click="$emit('click:button', item)"
+            >
+              {{ item }}
+            </SfButton>
+          </div>
+          <SfSidebar
+            :visible="isVisibleOnMobile && openSidebar"
+            :persistent="true"
+            :overlay="false"
+            class="sf-header__menu--sidebar"
+            @close="$emit('close')"
+          >
+            <SfMenuItem
+              v-for="(item, i) in menuItems"
+              :key="i"
+              :label="item"
+              class="sf-header__menu-item sf-header__menu-item--mobile"
+              @click="$emit('click:button', item)"
             />
-            <h1 v-else class="sf-header__title">{{ title }}</h1>
-          </SfLink>
+          </SfSidebar>
+          <div v-if="openContent" class="sf-header__menu-content">
+            <template v-for="item in menuItems">
+              <slot :name="item" />
+            </template>
+          </div>
         </slot>
-        <div class="sf-header__aside">
-          <!-- @slot Use this slot for language or currency selector -->
-          <slot name="aside" />
-        </div>
-        <div class="sf-header__actions">
-          <nav
-            class="sf-header__navigation"
-            :class="{ 'is-visible': isNavVisible }"
+      </nav>
+      <!--@slot Use this slot to replace default search bar-->
+      <slot
+        v-if="isVisible"
+        name="search"
+        v-bind="{ searchValue, searchPlaceholder }"
+      >
+        <SfSearchBar
+          :value="searchValue"
+          :placeholder="searchPlaceholder"
+          aria-label="Search"
+          class="sf-header__search"
+          @input="$emit('change:search', $event)"
+          @enter="$emit('enter:search', $event)"
+        />
+      </slot>
+      <!--@slot Use this slot to replace default header icons with custom content-->
+      <slot
+        name="header-icons"
+        v-bind="{
+          activeIcon,
+          cartIcon,
+          wishlistIcon,
+          accountIcon,
+        }"
+      >
+        <div v-if="isVisible" class="sf-header__icons">
+          <SfButton
+            v-if="accountIcon"
+            class="sf-button--pure sf-header__action"
+            data-testid="accountIcon"
+            @click="$emit('click:account')"
           >
-            <slot name="navigation"></slot>
-          </nav>
-          <!--@slot Use this slot to replace default search bar-->
-          <slot name="search" v-bind="{ searchValue, searchPlaceholder }">
-            <SfSearchBar
-              :value="searchValue"
-              :placeholder="searchPlaceholder"
-              aria-label="Search"
-              class="sf-header__search"
-              @input="$emit('change:search', $event)"
-              @enter="$emit('enter:search', $event)"
+            <SfIcon
+              :icon="accountIcon"
+              size="1.25rem"
+              :class="{
```

Comment:
For these kind of things, I think it is better to create a computed class, i.e.

```js
activeIconClass() {
  return this.activeIcon === 'account' ? 'sf-header__icon is-active' : null
} 
```

Reply:
I left this class object as more suitable for icons passing from loop over array.